components/search/ComboResults.vue

```diff
@@ -91,11 +91,9 @@ export default Vue.extend({
           return "No deck data (EDHREC)";
         }
 
-        if (combo.numberOfEDHRECDecks === 1) {
-          return "1 deck (EDHREC)";
-        }
-
-        return `${combo.numberOfEDHRECDecks} decks (EDHREC)`;
+        return `${combo.numberOfEDHRECDecks} deck${
+          combo.numberOfEDHRECDecks === 1 ? "" : "s"
+        } (EDHREC)`;
```

Comment:
This is fewer lines of code, but I find the previous way this displayed to be a lot more readable. Can you talk to me about the merits of changing it to this?

Reply:
I'm not really concerned with the brevity of the code. I think the largest thing it adds is consistency. I don't find value in doing things different ways in different places. Everywhere something is pluralized, it should follow the same pattern. This is largely how most internationalization libraries function so this is what I went to. I could extract this into a pluralization helper (as inspired by i18n libraries) if you prefer:

```js
const pluralize = (word, count, alternative) => count === 1 
  ? word
   : (alternative || word + 's')
 ;
```

Usage:

```js
`... ${helpers.pluralize('deck', combo.numerOfEDHRECDecks)} ...`
`... ${helpers.pluralize('box'. count, 'boxes')} ...`
```

Reply:
I'd be fine pulling this out into a pluralize helper method. Want to go that route for this PR?

If not, I can do it as a refactor after this gets merged.

Reply:
Is there already a location to place these helpers/utils?

Reply:
Let's leave that for a followup PR. I'm debating about whether it should just live in `lib` or be a plugin.